src/benchmarks/real-world/Microsoft.ML.Benchmarks/StochasticDualCoordinateAscentClassifierBench.cs

```diff
@@ -20,6 +21,7 @@ namespace Microsoft.ML.Benchmarks
     /// measure performance of making a single prediction from one of these pipelines, as
     /// well as making batch predictions on multiple rows of data from the same model.
     /// </summary>
+    [BenchmarkCategory(Categories.MachineLearning)]
```

Comment:
same as above: we don't need categories if there is only one category in given project

Reply:
See above.